Resources/views/FragmentAdmin/edit_one_association_script_fragment.html.twig

```diff
@@ -0,0 +1,51 @@
+{% autoescape false %}
```

Comment:
Avoid mixing js and twig please. Use a dedicated js files that finds data attributes in the DOM instead if you can.